src/Driver/SolverTypes/MISSolverType.jl

```diff
@@ -34,14 +34,23 @@ fast and slow dynamics respectively, depending on the state `Q`.
     Default: `MIS2`
 - `fast_method` (Function): Function defining the fast solver.
     Default: `LSRK54CarpenterKennedy`
-- `nsubsteps` (Int): Integer denoting the total number of times
+- `nsubsteps` (Tuple): Tuple denoting the total number of times
     to substep the fast process.
-    Default: `50`
+    Default: `(50,)`
 - `discrete_splitting` (Boolean): Boolean denoting whether a PDE level or
     discretized level splitting should be used. If `true` then the PDE is
     discretized in such a way that `f_fast + f_slow` is equivalent to
     discretizing the original PDE directly.
     Default: `false`
+- `hivi_splitting` (Boolean): Boolean denoting whether the fast processes need to
```

Comment:
We don't need `hivi_splitting`; there is already a `ClimateMachine.HEVISplitting` type that is passed as an argument to the MIS solver type constructor